src/util/entropyCreateStateFromJsons.ts

```diff
@@ -20,7 +20,7 @@ interface JsonAnnotation {
   end?: number
   start?: number
   segments?: JsonSegmentRange[]
-  strand: Strand
+  strand?: Strand
```

Comment:
A broader comment / question about TS as it relates to the JSON dataset. Beyond being semantically valid JSON anything could be present in the JSON (we don't do formal schema validation within Auspice). So should any types relating to the dataset contents therefore allow any and every field to be optional, and also any values to be `any` type? 

This relates to a7578d20f1c4db7b3755ae501816e9c11967a6d8, which proposes

```diff
-  if (typeof annotation.display_name === 'string') {
+  if (annotation.display_name !== undefined) {
```

(`annotation` here is direct from the JSON dataset). According to our schema `display_name` will always be unset or a string, but in reality it could be anything.